.github/workflows/release.yml

```diff
@@ -1,17 +1,17 @@
 name: Release
 on:
   push:
-    tags: ["v*"]
-
+    branches: [ master ]
+  release:
+    types: [ published ]
 jobs:
   build:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
+      - uses: olafurpg/setup-scala@v10
```

Comment:
Since I've opened the PR anyway -- I think we're using `olafurpg/setup-scala` for all the other project, so we might as well have it here too. Or was there a reason to go with `actions/setup-java`?

Reply:
I don't know what `setup-scala` gives us over `setup-java` which is working just fine - it's also used in SST. But why not.